src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,677 @@ description: Learn how to use images in Astro.
+
+Use the HTML `<img>` element when you cannot use the `<Image />` component, for example:
+  - for unsupported image formats, like `.svg`
+  - when you do not want your image optimized by Astro
+  - to access and change the `src` attribute dynamically client-side
+
+
+### Authorizing remote images
+
+You can configure lists of authorized image source URL domains and patterns for image optimization using [`image.domains`](/en/reference/configuration-reference/#imagedomains) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns). This configuration is an extra layer of safety to protect your site when showing images from an external source.
+
+Remote images from other sources will not be optimized, but using the `<Image />` component for these images will prevent Cumulative Layout Shift (CLS).
+
+For example, the following configuration will only allow remote images from `astro.build` to be optimized:
+
+```ts
+// astro.config.mjs
+export default defineConfig({
+  image: {
+    domains: ["astro.build"],
+  }
+});
+```
+
+The following configuration will only allow remote images from HTTPS hosts:
+
+```ts
+// astro.config.mjs
+export default defineConfig({
+  image: {
+    remotePatterns: [{ protocol: "https" }],
+  }
+});
+```
+
+## Using Images from a CMS or CDN
+
+Image CDNs work with all Astro image options. Use an image's full URL as the `src` attribute in the `<Image />` component, an `<img>` tag, or in Markdown notation. For image optimization with remote images, also [configure your authorized domains or URL patterns](#authorizing-remote-images).
+
+Alternatively, if the CDN provides a Node.js SDK, you can use that in your project. For example, [Cloudinary’s SDK](https://cloudinary.com/documentation/node_integration) can generate an `<img>` tag with the appropriate `src` for you.
+
+## Images in Markdown files
+
+Use standard Markdown `![alt](src)` syntax in your `.md` files. This syntax works with Astro's Image Services API to optimize your local images and authorized remote images.
 
 ```md
 <!-- src/pages/post-1.md -->
 
 # My Markdown Page
 
-<!-- Local image stored at public/assets/stars.png -->
-![A starry night sky.](/assets/stars.png)
-<img src="/assets/stars.png" alt="A starry night sky.">
+<!-- Local image stored in src/assets/ -->
+<!-- Use a relative file path or import alias -->
+![A starry night sky.](../assets/stars.png)
+
+<!-- Image stored in public/images/ -->
+<!-- Use the file path relative to public/ -->
+![A starry night sky.](/images/stars.png)
 
 <!-- Remote image on another server -->
-![Astro](https://docs.astro.build/assets/logomark-light.png)
-<img src="https://docs.astro.build/assets/logomark-light.png" width="25" alt="Astro">
+<!-- Use the full URL of the image -->
+![Astro](https://example.com/images/remote-image.png)
 ```
 
-### In MDX files
+The `<img>` tag is not supported, nor is the `<Image />` component.
```

Comment:
`<img>` does work in Markdown, but images won't be resolved or processed, much like in `.astro` files